third_party/docfx/templates/devsite/UniversalReference.common.js

```diff
@@ -102,7 +105,7 @@ function handleItem(vm, gitContribute, gitUrlPattern) {
   vm.sourceurl = common.getViewSourceHref(vm, null, gitUrlPattern);
 
   // set to null incase mustache looks up
-  vm.summary = vm.summary || null;
+  vm.summary = vm.summary != null ? vm.summary : null;

```

Comment:
How are the two different?

Reply:
This allows us to modify a summary that is an empty string. Without the distinction, we won't be able to add this sentence for pages contain an empty summary, but do have a ```summary``` key in the yaml.